addons/l10n_es_edi_tbai/data/template_LROE_bizkaia.xml

```diff
@@ -111,7 +111,7 @@
                             </t>
                         </CabeceraFactura>
                         <DatosFactura>
-                            <DescripcionOperacion t-out="invoice.ref"/>
+                            <DescripcionOperacion t-out="'%s: %s' % (invoice.name, invoice.ref)"/>
```

Comment:
I'm not sure we really want the `invoice.name` here for vendor bills.

It's common in Spain to renumber account moves before declaring taxes, to avoid gaps. Of course, customer invoices cannot be renumbered. But other kind of moves can.

Thus, if this is notifying instantly a vendor bill, but the `invoice.name` can change later, the field contents would become useless or misleading.

If you want to identify the invoice in our DB, I guess it'd be better to put another thing such as the `invoice.id`. Otherwise, just keep this as it was (`invoice.ref`):

```suggestion
                            <DescripcionOperacion t-out="'ID %d: %s' % (invoice.id, invoice.ref)"/>
```

Reply:
It is just that an id is not that practical.

Reply:
Then you can revert this change and leave just the ref when it's a vendor bill.